File: app/src/main/java/com/softdev/smartraysam/carfeeds/model/CarFeedsResponse.java
package com.softdev.smartraysam.carfeeds.model;

import retrofit2.Call;
import retrofit2.http.GET;



/**
 * Created by Smart Raysam on 15-09-2018.
 */

public interface CarFeedsResponse {
    @GET("/wunderbucket/locations.json")
    Call<Cars> getCars();
}

File: app/src/main/java/com/softdev/smartraysam/carfeeds/model/CarFeedsResponse.kt
package com.softdev.smartraysam.carfeeds.model

import retrofit2.Call
import retrofit2.http.GET


/**
 * Created by Smart Raysam on 15-09-2018.
 */

interface CarFeedsResponse {
    @get:GET("/wunderbucket/locations.json")
    val cars: Call<Cars>
}
